include/stout/internal/windows/attributes.h

```diff
@@ -15,6 +15,12 @@
 #include <string>
 
 #include "stout/error.h"
+// Since 'fmt' library doesn't support 'wide' conversion (e.g
+// from 'std::wstring' to 'std::string' and vice versa) we use
+// API from 'include/stout/stringify.h' (e.g:
+// std::string stringify(const std::wstring& wstr) - function).
+// Check the issue for fmt conversion on github:
+// https://github.com/fmtlib/fmt/issues/1116
```

Comment:
It looks like this issue is closed: is this still the right issue to link to? (here and below)

Reply:
I think yep because in this link there is important discussion about wide string conversion for fmt lib.